File: C#/Programming Basics with C#/12. Nested Loops/07. Name Wars/NameWars.cs
using System;
using System.Text;

class NameWars
{
    static void Main()
    {
        int sum = 0;
        int sum1 = 0;
        string winner = " ";

        for (string name = Console.ReadLine(); name != "STOP"; name = Console.ReadLine())
        {

            byte[] asciiBytes = Encoding.ASCII.GetBytes(name);
            for (int i = 0; i < asciiBytes.Length; i++)
            {
                sum = sum + asciiBytes[i];
            }
            if (sum > sum1)
            {
                winner = name;
                sum1 = sum;
            }
            sum = 0;
        }
        Console.WriteLine($"Winner is {winner} - {sum1}!");
    }
}

File: C++/Programming Basics with C++/12. Nested loops/simple_tasks/NameWars.cpp
#include<iostream>

using namespace std;

int main() {
    int sum = 0;
    int sum1 = 0;
    string winner;
    string name;
    getline(cin, name);

    while (name != "STOP") {
        for (int i = 0; i < name.length(); i++) {
            sum += name[i];
        }

        if (sum > sum1) {
            winner = name;
            sum1 = sum;
        }

        sum = 0;
        getline(cin, name);
    }

    cout << "Winner is " << winner << " - " << sum1 << "!" << endl;

    return 0;
}